src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
+import { Comment } from '../../react-app-env';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  const commentsList = comments.map(comment => (
```

Comment:
move map logic inside the return statement, don't create separate variable for this